diff -ruN lwtools-4.18/Makefile lwtools-4.18.safe/Makefile
--- lwtools-4.18/Makefile	2021-01-19 21:08:37.000000000 -0500
+++ lwtools-4.18.safe/Makefile	2021-03-14 15:56:54.000000000 -0400
@@ -49,10 +49,7 @@
 MAIN_TARGETS := lwasm/lwasm$(PROGSUFFIX) \
 	lwlink/lwlink$(PROGSUFFIX) \
 	lwar/lwar$(PROGSUFFIX) \
-	lwlink/lwobjdump$(PROGSUFFIX) \
-	lwcc/lwcc$(PROGSUFFIX) \
-	lwcc/lwcc-cpp$(PROGSUFFIX) \
-	lwcc/lwcc-cc$(PROGSUFFIX)
+	lwlink/lwobjdump$(PROGSUFFIX)
 
 LWCC_LIBBIN_FILES = lwcc/lwcc-cpp$(PROGSUFFIX) lwcc/lwcc-cc$(PROGSUFFIX)
 LWCC_LIBLIB_FILES =
@@ -61,6 +58,9 @@
 .PHONY: all
 all: $(MAIN_TARGETS)
 
+package: $(MAIN_TARGETS)
+	zip -j lwtools-$(PACKAGE_VERSION).zip $^
+
 lwar_srcs := add.c extract.c list.c lwar.c main.c remove.c replace.c
 lwar_srcs := $(addprefix lwar/,$(lwar_srcs))
 
